internal/flags.go

```diff
@@ -328,24 +334,41 @@ func PopulateFlags(c *cli.Context) (ret *FlagStorage) {
 		HTTPTimeout:  c.Duration("http-timeout"),
 
 		// Common Backend Config
+		Backend:        c.String("storage-backend"),
 		Endpoint:       c.String("endpoint"),
 		UseContentType: c.Bool("use-content-type"),
 
 		// Debugging,
 		DebugFuse:  c.Bool("debug_fuse"),
 		DebugS3:    c.Bool("debug_s3"),
 		Foreground: c.Bool("f"),
+
```

Comment:
do we need this?

Reply:
Yes, I have added all the flags user can pass as args to one structure